### ISSUE TITLE
DynComp failures with tests/daikon-tests/RatPoly

### ISSUE THREAD

**Issue by mernst** [codespecs/daikon#26]
Originally reported on Google Code with ID 26
```
If you cd to tests/daikon-tests/RatPoly; then do:
  make clean
  make dyndomp-diff

The four output files:
  RatPoly.txt-cset
  RatPoly.txt-cset.diff
  RatPoly.decls-DynComp
  RatPoly.decls-DynComp.diff

show many differences.  We looked at one in particular from RatPoly.txt-cset.diff:

 Variable sets for PolyCalc.RatPoly.findTermIndex(PolyCalc.RatTermVec ts, I expt) exit
-  [1] [Parameter expt]
-  [1] [return]
+  [2] [Parameter expt, return]
   [1] [Parameter ts]
   [1] [Field ts.wrapped]
   [1] [List ts.wrapped[..]]
   [1] [Class of ts.wrapped[..]]
We didn't think that Parameter expt and return should be in the same set.


```

Reported by `markro@cs.washington.edu` on 2014-02-05 17:08:03

**Comment by markro49** [comment]
Interesting note:  if you use --no-jdk then you get the expected result.   In the past, I noticed something similar with java.util.Random, but have not been able to find a similar cause as yet.

**Comment by markro49** [comment]
Decided not to use the instrumented version of Random as the method generates values based on an initial seed value.  This has the undesirable side effect of putting all the generated values in the same comparison set when they should be distinct.
fixed with e16e5ad9fc1399f7daf77ddf67f74e9a747445ea
